src/main/java/org/symphonyoss/symphony/messageml/markdown/nodes/form/CheckboxNode.java

```diff
@@ -0,0 +1,30 @@
+package org.symphonyoss.symphony.messageml.markdown.nodes.form;
+
+/**
+ * Class that Represents a Markdown Node for the "Checkbox" form element.
+ *
+ * @author Lucas Macedo
+ * @since 10/21/2019
+ */
+public class CheckboxNode extends FormElementNode {
+  private final static String MARKDOWN = "Checkbox";
+
+  private String label;
+  
+  public CheckboxNode() {
+  }
+
+  public CheckboxNode(String label) {
+    this.label = label;
+  }
+
+  @Override
+  public String getOpeningDelimiter() {
```

Comment:
You don't need to override this method on any of the Node classes you did it. It already does when you're using a FormElementNode. Also, the constructor should call super() with the markdown tag